### ISSUE TITLE
Editing YAML

### ISSUE THREAD

**Issue by aloneguid** [Azure/service-fabric-mesh-preview#321]
This is not exactly an issue, mostly a question. Which tools are you using to edit YAML files? So far I've been really struggling to make a valid YAML, especially when resource description grows. There is always a case of mistyped space or extra delimiter and so on. Both VS and VS Code is lacking a good yaml editing experience too. I understand that any valid JSON document is a valid YAML file, so I guess I could revert to JSON in descriptions. Unless I'm really missing something here?

**Comment by tomkerkhove** [comment]
I use visual studio code with this extension - https://marketplace.visualstudio.com/items?itemName=adamvoss.yaml

**Comment by SamirFarhat** [comment]
True, i had several problems with YAML, formatting is not one of its good points.

**Comment by ChackDan** [comment]
I use VS code too..

**Comment by guibirow** [comment]
For VSCode this one is really good: https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml

![image](https://user-images.githubusercontent.com/943026/50010374-30557000-ffb1-11e8-9301-1893624f5035.png)
